src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Services/DataSetVersionMappingService.cs

```diff
@@ -63,6 +63,21 @@ public async Task<Either<ActionResult, Unit>> CreateMappings(
             sourceFilterMeta,
             nextVersionMeta.Filters);
 
+        var hasDeletedIndicators = await HasDeletedIndicators(
```

Comment:
I guess that post-MVP, when we have the chance to map Indicators, we can retire this column.

Reply:
Yep I would think so :smile: